### PR TITLE
Bump libpqxx to v7.7.5

### PR DESCRIPTION
**Pull request by rschu1ze**
This is part of fixing #67069 and the sister PR of #69564.

libpqxx was updated from v7.6.0 (a three year old version) to v7.7.5. This is not the the most recent version as newer versions broke some tests: https://github.com/ClickHouse/ClickHouse/pull/69572). Need to check that issue separately.

### Changelog category (leave one):
- Not for changelog (changelog entry is not required)

### CODE REVIEW

**Comment by robot-clickhouse-ci-1** [comment]
*This is an automated comment for commit aab0d3dd9ea4290f955946a806fe2fd7c4f21f02 with description of existing statuses. It's updated for the latest CI running*

[✅ Click here](https://s3.amazonaws.com/clickhouse-test-reports/69580/aab0d3dd9ea4290f955946a806fe2fd7c4f21f02/ci_running.html) to open a full report in a separate page

<details><summary>Successful checks</summary>
<table>
<thead><tr><th>Check name</th><th>Description</th><th>Status</th></tr></thead>
<tbody>
<tr><td>Docs check</td><td>Builds and tests the documentation</td><td><a href="https://s3.amazonaws.com/clickhouse-test-reports/0/f62cb32c5c9f94e4d661b6a00c012564af5200f7/docs_check.html">✅ success</a></td></tr>
<tbody>
</table>
</details>

File: contrib/libpqxx-cmake/CMakeLists.txt [hunk 1]
```diff
@@ -19,59 +19,25 @@ set (SRCS
...
-    "${LIBRARY_DIR}/src/row.cxx"
-    "${LIBRARY_DIR}/src/params.cxx"
     "${LIBRARY_DIR}/src/util.cxx"
     "${LIBRARY_DIR}/src/version.cxx"
+    "${LIBRARY_DIR}/src/wait.cxx"
 )
 
-# Need to explicitly include each header file, because in the directory include/pqxx there are also files
-# like just 'array'. So if including the whole directory with `target_include_directories`, it will make
-# conflicts with all includes of <array>.
```
Comment (davenger) [inline contrib/libpqxx-cmake/CMakeLists.txt:39]: Is this concern no longer valid?

**Comment by rschu1ze** [comment]
Stateless tests (release): https://github.com/ClickHouse/ClickHouse/issues/69624

**Comment by rschu1ze** [comment]
@davenger Maybe you like to approve? Thanks.

**Review by davenger — APPROVED** [review]

**Pull request merged**